docs/config.rst

```diff
@@ -0,0 +1,184 @@
+Configuration Handling
+======================
+
+Application Configuration Values
+----------------------------
+
+The following configuration values are used internally by Datacube-explorer:
+
+
+.. py:data:: CUBEDASH_CORS
+
+    Enable Cross Origin Resource Sharing (CORS) for ``stac`` and ``api``.
+
+    Default: ``True``
+
+.. py:data:: CUBEDASH_PRODUCT_GROUP_BY_REGEX
+
+    Tuple containing regex for product name matching and group name
+
+    Default: ``None``
+    Example: ``((r'^usgs_','USGS products'), (r'_albers$','C2 Albers products'), (r'level1','Level 1 products'), )``
```

Comment:
@SpacemanPaul , the default is `none`, but i'd like to give example, does this explanation look ok?

Reply:
Makes sense to me.